### PR TITLE
feat: change integrator string for gas tab

### PR DESCRIPTION
**Pull request by tcheee**

### CODE REVIEW

File: src/components/Widgets/Widget.tsx [hunk 1]
```diff
@@ -37,6 +37,8 @@ export function Widget({ starterVariant }: WidgetProps) {
   const wagmiConfig = useConfig();
   const { isMultisigSigner, getMultisigWidgetConfig } = useMultisig();
   const { multisigWidget, multisigSdkConfig } = getMultisigWidgetConfig();
+  const isGasVariant =
+    window?.location?.href?.includes(TabsMap.Refuel.variant) ?? false;
```
Comment (dennyscode) [inline src/components/Widgets/Widget.tsx:41]: Another approach could be: "activeTab" from store. Could give that a try but not sure which one works better.

File: .env.development [hunk 2]
```diff
@@ -3,6 +3,7 @@ VITE_HOTJAR_ID=3307684
 VITE_HOTJAR_SNIPPET_VERSION=6
 VITE_LIFI_API_URL=https://develop.li.quest/v1
 VITE_WIDGET_INTEGRATOR=dev.jumper.exchange
+VITE_WIDGET_INTEGRATOR_REFUEL=dev.jumper.exchange.gas
```
Comment (dennyscode) [inline .env.development:6]: Also update the env declaration file accordingly.

**Review by dennyscode — CHANGES_REQUESTED** [review]

File: .env.development (hunk 2 above)
Comment (tcheee) [reply .env.development:6]: Done thanks

File: src/components/Widgets/Widget.tsx (hunk 1 above)
Comment (tcheee) [reply src/components/Widgets/Widget.tsx]: Way cleaner, thank you!

**Review by dennyscode — APPROVED** [review]

**Review by Addminus — APPROVED** [review]

**Pull request merged**